12/calculator/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
<img width="502" alt="Screenshot 2022-10-30 at 20 15 50" src="https://user-images.githubusercontent.com/34886027/198894808-387c34b1-ab9b-4d79-ae52-2e6d4b34101d.png">
<img width="502" alt="Screenshot 2022-10-30 at 20 15 32" src="https://user-images.githubusercontent.com/34886027/198894809-0fc155bb-09e1-46c8-9189-db83607dd0e0.png">

`0.1 + 0.2 = 0.3`
Only 1 comma is possible in numbers